Header header

uint32 num_beams
uint32 range_radius
float32 angular_view

float64[] d_obs

float64 longitude
float64 latitude
float64 yaw
